app/models/folio/item.rb

```diff
@@ -184,6 +184,16 @@ def aeon_site
       permanent_location.details['pageAeonSite']
     end
 
+    def self.call_number(dyn)
+      call_number = dyn.dig('effectiveCallNumberComponents', 'callNumber')
+      call_number unless call_number == 'UNCLASSIFIED'
+
```

Comment:
But shelved by titles are unclassified right? Basically the code doesn't dig into location data unless there is a call number that is unclassified.

Reply:
I'm not sure when things are given the call number "UNCLASSIFIED" (there's ~305 of them), but it's not the same as shelved-by-title. Some items have that call number and are in a regular location (why? I don't know): https://searchworks.stanford.edu/view/406193

There's also a bunch of things that are shelved-by-title, but have a variety of call numbers, e.g. there's a bunch that were given LC call numbers, but they're in `SCI-SHELBYSERIES`:
https://searchworks.stanford.edu/?search_field=search&q=location_code_facet_ssim%3ASCI-SHELBYSERIES

And there's a bunch of items in a SHELBYTITLE location with "temporary" call numbers (e.g. [10773405](https://requests.stanford.edu/pages/new?item_id=10773405&origin=MARINE-BIO&origin_location=MAR-SHELBYTITLE)) that we definitely don't want to show (Searchworks is currently showing them because we were looking for SHELBYTITL (Symphony), not SHELBYTITLE (FOLIO))

Reply:
Just to make sure I understand we want to display shelved by title for everything that has an effective location that is shelved by title, even if there is a call number?

Reply:
I think parity would match the call number we're indexing for Searchworks:
https://github.com/sul-dlss/searchworks_traject_indexer/blob/main/lib/traject/config/folio_config.rb#L2091-L2099

(mashing up "Shelved by title" with the volume/enumeration/etc information)

Reply:
Okay. I have updated to match the indexer.